sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp

```diff
@@ -239,9 +239,9 @@ namespace Azure { namespace Storage { namespace Blobs {
      *
      * @param options Optional parameters to execute this function.
      * @param context Context for cancelling long running operations.
-     * @return A GetBlobContainerAccessPolicyResult describing the container's access policy.
+     * @return A BlobContainerAccessPolicy describing the container's access policy.
      */
-    Azure::Response<Models::GetBlobContainerAccessPolicyResult> GetAccessPolicy(
+    Azure::Response<Models::BlobContainerAccessPolicy> GetAccessPolicy(
```

Comment:
Needs a changelog entry.

The one you had previously works well:
https://github.com/Azure/azure-sdk-for-cpp/pull/1923/files#diff-8577e40569d1adddc6b285cef21ee4b8342f9cac9c442ceb6e62c7951c9057b3L21